scripts/dev

```diff
@@ -403,6 +403,7 @@ namespace :gql do
     task :yarn do
       ENV["FORCE_COLOR"] = "true"
 
+      sh "rm -r ./src/queries/types"
```

Comment:
Interestingly, our `pre-commit` check doesn't actually run `scripts/dev gql`, but [actually runs `yarn generate` manually](https://github.com/CMSgov/easi-app/blob/3580ce11387ce9ec41698050cc4c25632ac34349/scripts/pre-commit-check-code-generation#L12-L14). I think we'll need to make a change in order for this to work in every instance that we call GQL generation, and I see 2 potential avenues of addressing this:

1. Update `scripts/pre-commit-check-code-generation` to call `scripts/dev gql` instead of `go generate` and `yarn generate`
2. Instead of adding this line here, add it to [`package.json`'s `generate`](https://github.com/CMSgov/easi-app/blob/3580ce11387ce9ec41698050cc4c25632ac34349/package.json#L95) command directly, that way if you call `yarn generate` yourself (or in the aforementioned script), the types are cleared.

Reply:
Awesome! Thanks Clay, I appreciate you pointing that out.

I have made updates to this PR. Please let me know what you think!